src/commands/build.ts

```diff
@@ -45,11 +45,12 @@ export class BuildCommand implements Command {
           'used for service worker generation.'
     },
     {
-      name: 'insert-dependency-links',
+      name: 'add-prefetch',
       type: Boolean,
-      description: 'Flatten dependency tree downloads by inserting ' +
-          'additional `<link rel="prefetch">` tags into ' +
-          'entrypoint and `<link rel="import">` tags into fragments and shell.'
+      description: 'Add dependency tree prefetching by inserting ' +
```

Comment:
I'd remove "tree", we haven't used the term "dependency tree" before. Maybe:

"Enable dependency prefetching by inserting..."